jcstress-core/src/main/java/org/openjdk/jcstress/Options.java

```diff
@@ -82,7 +81,8 @@ public boolean parse() throws IOException {
         OptionSpec<String> parse = parser.accepts("p", "Re-run parser on the result file. This will not run any tests.")
                 .withRequiredArg().ofType(String.class).describedAs("result file");
 
-        OptionSpec<Boolean> list = parser.accepts("l", "List the available tests matching the requested settings.")
+        OptionSpec<Boolean> list = parser.accepts("l", "List the available tests matching the requested settings, " +
+                        "after all filters (like CPU count) are applied. In verbose mode it prints all real combinations which will run.")
```

Comment:
"List the available tests matching the requested settings, " +
                        "after all filters (like CPU count) are applied. In verbose mode it prints all real combinations which will run."

this might have been done for better readability, but remove the string concatenation, define it as a part of single string, possibly at class level by making it static final

Reply:
fixed